docs/source/installing_docker.rst

```diff
@@ -132,6 +128,9 @@ To upgrade running Kiwi TCMS containers execute the following commands::
     docker-compose up -d
     docker exec -it kiwi_web /Kiwi/manage.py migrate
 
+The migrations may be performed by using the web-based tool:
+``https://<your_domain_or_ip>/init-db/``
+
```

Comment:
Remove this block. We don't want to recommend performing web based migrations for upgrades.